ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -66,14 +66,38 @@ export interface CanvasCourseSection extends CanvasCourseSectionBase {
   total_students: number
 }
 
-export enum UserEnrollmentType {
+export enum CanvasRoles {
```

Comment:
`CanvasRole`?  Minor thing, but I think it's typical to use the singular form for enums. For example, see `Direction` and `LogLevel` in the TypeScript documentation: https://www.typescriptlang.org/docs/handbook/enums.html